CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## Supported Release 2.3.0
+### Summary
+A release containing many bugfixes with additional features, including a modulesync update.
+
+#### Features
+- Apt_updates facts now use /usr/bin/apt-get.
+- Addition of notify update to apt::source.
+- Update to newest modulesync_configs.
+- Installs software-properties-common for xenial.
+- Modulesync updates.
```

Comment:
Do we need to mention module sync twice? Most people have no idea what it is / don't really care.